catalog/datasets/wikidata-entities.jsonld

```diff
@@ -10,7 +10,7 @@
   ],
   "creator": [
     {
-      "@id": "https://www.wikidata.org"
+      "@id": "http://www.wikidata.org/entity/Q2013"
```

Comment:
I had to change this IRI to disambiguate it from the dataset, which already has IRI `https://www.wikidata.org`. I picked the Wikidata organization on Wikidata; better ideas are welcome.

Reply:
Or `https://www.wikidata.org/entity/Q180`, to refer to the creator of the dataset? `http://www.wikidata.org/entity/Q2013` seems to be the dataset itself?

Reply:
Yes, much better!